manifests/config.pp

```diff
@@ -11,33 +11,35 @@
 # Sample Usage:
 #
 class activemq::config (
-  $server_config,
-  $instance,
-  $package,
-  $path = '/etc/activemq/activemq.xml',
-  $server_config_show_diff = 'UNSET',
+  $server_config           = $activemq::server_config_content,
+  $instance                = $activemq::instance,
+  $package                 = $activemq::package,
```

Comment:
This one (`$package`) is useless as it is never used, either keep it and let the `require => Package[$package]` or remove it and then you can change the require as below. Personally I tend to prefer the first solution